core/testing.go

```diff
@@ -1,48 +1,26 @@
 package core
 
 import (
-	"fmt"
 	"net"
 	"net/url"
 	"testing"
+	"time"
 
-	appconfig "github.com/cosmos/cosmos-sdk/server/config"
-	"github.com/stretchr/testify/require"
 	tmconfig "github.com/tendermint/tendermint/config"
 	tmrand "github.com/tendermint/tendermint/libs/rand"
-	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
 
 	"github.com/celestiaorg/celestia-app/test/util/testnode"
 )
 
-// TestConfig encompasses all the configs required to run test Tendermint + Celestia App tandem.
-type TestConfig struct {
-	ConsensusParams *tmproto.ConsensusParams
-	Tendermint      *tmconfig.Config
-	App             *appconfig.Config
-
-	Accounts     []string
-	SuppressLogs bool
-}
```

Comment:
This struct is effectively a subset of the new `testnode.Config`, so replacing with just that config

Reply:
Did you add it recently? IIRC there was not such functionality before

Reply:
in the last refactor yeah

everything is in the testnode.Config now

Reply:
Cool, hope the nodes code helped there